vendor/k8s.io/kubernetes/pkg/kubectl/cmd/version/version.go

```diff
@@ -106,7 +107,7 @@ func (o *VersionOptions) Run() error {
 		versionInfo   Version
 	)
 
-	clientVersion := getKubectlVersion()
+	clientVersion := version.Get()
```

Comment:
Why are we patching this?

Reply:
we were patching, this is removing the patch.